.devcontainer/Dockerfile

```diff
@@ -1,8 +1,26 @@
-FROM node:lts
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.158.0/containers/typescript-node/.devcontainer/base.Dockerfile
 
-WORKDIR /workspaces
+# [Choice] Node.js version: 14, 12, 10
+ARG VARIANT="14-buster"
```

Comment:
You set the variant in devcontainer.json, this is not needed.

Reply:
Without this it wouldn't build 🤷 

It's also what they have here https://github.com/microsoft/vscode-dev-containers/blob/v0.158.0/containers/typescript-node/.devcontainer/devcontainer.json

Reply:
right :facepalm: , then we should remove it from devcontainer.json, there is no point in having it in both places, that's only useful if you use the same Dockerfile against multiple devcontainer configurations.

Reply:
Done.